controller/volume_controller.go

```diff
@@ -807,6 +807,7 @@ func (vc *VolumeController) createReplica(v *longhorn.Volume) (*longhorn.Replica
 		replica.Spec.RestoreName = backupID
 	}
 	replica.Spec.DataPath = longhornDirectory + "/replicas/" + v.Name + "-" + util.RandomID()
+	replica.Spec.BaseImage = v.Spec.BaseImage
```

Comment:
Put this line in the last of replica definition instead.

In fact, both of those lines should be put there...